ci/kind/kind-setup.sh

```diff
@@ -22,6 +22,8 @@ CLUSTER_NAME=""
 ANTREA_IMAGE="antrea/antrea-ubuntu:latest"
 IMAGES=$ANTREA_IMAGE
 ANTREA_CNI=false
+UNTIL=false
+TIME=""
```

Comment:
I don't think you need `UNTIL`, as you can check whether `TIME` is non-empty

`TIME` is also too generic of a name